src/architecture/services.md

```diff
@@ -293,9 +293,9 @@ Here is a list of things to figure out when developing an Exonum service:
   should be a part of the local configuration or stored in the blockchain.)
 
 !!! tip
-    [The cryptocurrency tutorial](../get-started/create-service.md)
-    provides a hands-on guide how to build an Exonum service that implements
-    a minimalistic crypto-token.
+    The [How to Create a Service](../get-started/create-service.md)
+    tutorial provides a hands-on guide how to build an Exonum service that
+    implements a minimalistic crypto-token.
```

Comment:
I know that our team do not like the word "token", so probably we'd better changed it for "cryptocurrency"